packages/validator/src/utils/action.ts

```diff
@@ -0,0 +1,36 @@
+import { insertValue, replaceParamIndexWithName, replacePathsWithValues } from './utils';
+import { Roots } from '../types';
+
+export function execute(specs: any, template: any, currentPath: string[], roots: Roots) {
```

Comment:
Can we replace `any` with `object` ? (It will at least carry a better type name)

Reply:
But I think we can type this even better no?

Reply:
> Can we replace any with object ? (It will at least carry a better type name)

no, action can be executed on primitives as well

> But I think we can type this even better no?

I don't think so, any suggestions?